### DISCUSSION TITLE
I can't draw

### DISCUSSION THREAD

**Discussion by HinKSH** [BenJeau/react-native-draw#72] [Q&A]
I just follow your instruction, however, I can't draw and have error of cannot find the variable t.
<img width="655" alt="螢幕截圖 2022-11-09 下午2 30 17" src="https://user-images.githubusercontent.com/105152713/200756030-a53f56c9-0b9c-4a96-ae71-f4684d2b3963.png">

**Comment by LokCheuk** [comment]
same here

**Comment by LokCheuk** [comment]
@BenJeau can you help us out(?)

**Comment by AbdouRGitHub** [comment]
Hello. I have the same issue that i'm trying to resolve. Do you find the solution ? Thank You
